src/twisted/plugins/twisted_reactors.py

```diff
@@ -4,59 +4,70 @@
 from __future__ import absolute_import, division
 
 from twisted.application.reactors import Reactor
-from twisted.python.compat import _PY3
+from twisted.python.reflect import requireModule
 
-default = Reactor(
+def _addReactorToAll(shortName, moduleName, description):
+    """
+
+    @param shortName: Short name of reactor
+    @param moduleName: The fully-qualified module name of the reactor
+    @param description: Description of reactor
+    @return: newly created reactor
+    @rtype: L{Reactor} or L{None}
+    """
+    newReactor = None
+    if requireModule(moduleName) is not None:
+        newReactor = Reactor(shortName, moduleName, description)
+        __all__.extend([shortName])
+    return newReactor
+
+
+__all__ = []
```

Comment:
Minor stylistic tweak: I think it's better style to declare a variable above any of its uses; since `_addReactorToAll` modifies `__all__` it would be nice to declare it above.